cas-server-core-authentication/src/main/java/org/jasig/cas/authentication/DefaultAuthenticationTransactionManager.java

```diff
@@ -15,13 +13,11 @@
  * @since 4.2.0
  */
 @Component("defaultAuthenticationTransactionManager")
-public final class DefaultAuthenticationTransactionManager implements AuthenticationTransactionManager {
+public class DefaultAuthenticationTransactionManager implements AuthenticationTransactionManager {
 
     private static final Logger LOGGER = LoggerFactory.getLogger(DefaultAuthenticationTransactionManager.class);
 
-    @NotNull
-    @Autowired
-    @Qualifier("authenticationManager")
+    @Resource(name="authenticationManager")
```

Comment:
Just a note on the unrelated work I'm doing on another - I do the opposite i.e. replacing the use of `@Resource` with `@Autowired @Qualifier` combination to be consistent throughout our config infrastructure